Dockerfile

```diff
@@ -29,9 +28,10 @@ RUN apt-get update &&\
     add-apt-repository -y ppa:deadsnakes/ppa &&\
     apt install -y python3.7 \
                    python3.7-dev &&\
-    # this relinks pip to python3.7 
-    python3.7 -m pip install --upgrade pip &&\
+    curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py &&\
```

Comment:
would be best if you could follow up and delete `get-pip.py`

Reply:
Oh, my bad. Will do.

Reply:
Should be fine now.